screens/Profile/profileStyles.ts

```diff
@@ -36,12 +36,14 @@ export const profileStyles = StyleSheet.create({
         borderColor: lightPeach,
     },
     profileContainer: {
+        flex: 1,
+        marginTop: "35%",
         paddingHorizontal: "2.5%",
-        width: "100%"
     },
     topBackground: {
         backgroundColor: lightPeach,
         height: "20%",
+        position: "absolute",
```

Comment:
Could you explain the reasoning why you added a position: "absolute"? What was the problem with the layout before?

Reply:
Sadly the database is not working on my end for now so I'll show you on the figma. This is in order to get this overlap of the profile picture over the top background.

<img width="272" alt="Capture" src="https://github.com/uniconnect-epfl/uniconnect/assets/93329823/b2d90b89-d736-454a-91aa-46a8e9bf9b4e">

Reply:
I see. Nice approach!